Makefile

```diff
@@ -14,4 +14,9 @@ format:
 	cargo fmt -- --write-mode overwrite
 
 clean:
-	cargo clean
\ No newline at end of file
+	cargo clean
+
+examples: test
```

Comment:
`cargo test` compiles examples but `cargo build` does not. [manifest.html#examples](http://doc.crates.io/manifest.html#examples)